package.json

```diff
@@ -34,12 +34,13 @@
   "dependencies": {
     "@stablelib/random": "^1.0.1",
     "cids": "^1.1.6",
-    "dag-jose-utils": "^0.1.1",
+    "dag-jose-utils": "^1.0.0",
     "did-jwt": "^5.6.1",
     "did-resolver": "^3.1.0",
+    "multiformats": "^9.4.10",
+    "query-string": "^7.0.0",
     "rpc-utils": "^0.3.4",
-    "uint8arrays": "^2.1.5",
-    "query-string": "^7.0.0"
+    "uint8arrays": "^2.1.5"
```

Comment:
@stephhuynh18 why not update uint8arrays and other packages as well here?

Reply:
I just did the ones listed in the zenhub ticket but i can update them all